packages/web-react/src/components/FileUploader/README.md

```diff
@@ -422,6 +422,7 @@ The rest of the properties are created from the default `<ul>` element. [More ab
 | `onDismiss`        | `(key: string) => FileQueueMapType`  | —        | ✔        | Callback to delete an item from the queue                                   |
 | `onEdit`           | `(event: Event, file: File) => void` | —        | ✕        | Show and add function to edit button                                        |
 | `onError`          | `FileUploaderErrorCallbackType`      | —        | ✕        | Callback on error condition                                                 |
+| `objectFit`        | [`cover` \| `contain`]               | `cover`  | ✕        | CSS value for image preview in FileUploaderAttachment                       |
```

Comment:
so cover is default! :) great :))

Reply:
Anyway I would improve the description to something like:
> Defines FileUploaderAttachment image fit in container

Reply:
you are right, thats better description, thank you